task/python-sdk-azure/0.1/python-sdk-azure.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: python-azure-sdk
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.54.0"
+    tekton.dev/categories: cloud
```

Comment:
@piyush-garg @PuneetPunamiya is hub case sensitive while parsing the category? `Cloud` vs `cloud`, former one is what is defined in hub and later one is defined here.